### PR TITLE
release: jdk11.0.18+10

### PR DESCRIPTION
**Pull request by steelhead31**
shipped:
x64 linux
aarch64 linux
ppc64le
arm32
s390x
alpine

### CODE REVIEW

File: linux/jdk/alpine/src/main/packaging/temurin/11/APKBUILD [hunk 1]
```diff
@@ -89,7 +89,7 @@ _jdk() {
 }
 
 sha256sums="
-774d5955c09893dda14e3eb0fd3e239a6b2cec58615fcf4ec68747260b6e1cc1  OpenJDK11U-jdk_x64_alpine-linux_hotspot_$_pkgver.tar.gz
+81e51ec2da61b227c522881313c7e2a090b29d6cc4171b5e1db3f3d17d863e44  OpenJDK11U-jdk_x64_alpine-linux_hotspot_$_pkgver.tar.gz
```
Comment (zdtsw) [inline linux/jdk/alpine/src/main/packaging/temurin/11/APKBUILD:92]: 42722bdbee99ad1430d6e88f17608909f26b57ab7761fb6b87ff4dbf6a8928bc

Comment (steelhead31) [reply linux/jdk/alpine/src/main/packaging/temurin/11/APKBUILD]: You know, im 99% sure I changed this..!

Comment (steelhead31) [reply linux/jdk/alpine/src/main/packaging/temurin/11/APKBUILD]: You know, im 99% sure I changed this..!

**Review by gdams — APPROVED** [review]

**Review by zdtsw — APPROVED** [review]

File: linux/jdk/alpine/src/main/packaging/temurin/11/APKBUILD (hunk 1 above)
Comment (zdtsw) [reply linux/jdk/alpine/src/main/packaging/temurin/11/APKBUILD]: i know you changed it, just you changed to the wrong sha :D

**Pull request merged**